### PR TITLE
[tests-only] added tests for DELETE request endpoints with body

### PR DESCRIPTION
**Pull request by kiranadh1452**
## Description
This PR adds the test for the `DELETE` request to endpoints with body for `oCIS` backend.

## Related Issue

- Part of https://github.com/owncloud/core/issues/40144

## Motivation and Context

https://datatracker.ietf.org/doc/html/rfc4918#section-8.4
If a body is sent, then the server is supposed to return HTTP status 415 (Unsupported Media Type).

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by SwikritiT — COMMENTED** [review]
LGTM

File: tests/acceptance/features/apiAuthWebDav/webDavDELETEAuthOC10Issue40144.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    Then the HTTP status code of responses on all endpoints should be "204"
+
+  @skipOnOcV10 @personalSpace
+  Scenario: send DELETE requests to webDav endpoints with body as normal user using the spaces WebDAV API
+    When user "Alice" requests these endpoints with "DELETE" including body "doesnotmatter" about user "Alice"
+      | endpoint                                           |
+      | /remote.php/dav/spaces/%spaceid%/textfile0.txt     |
+      | /remote.php/dav/spaces/%spaceid%/PARENT/parent.txt |
+      | /remote.php/dav/spaces/%spaceid%/PARENT            |
+    Then the HTTP status code of responses on all endpoints should be "204"
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavDELETEAuthOC10Issue40144.feature:33]:
```suggestion
```
This scenario is space related and only runs for oCIS. No need to put this scenario in oC10 bug demo.

File: tests/acceptance/features/apiAuthWebDav/webDavDELETEAuthOC10Issue40144.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@api
```
Comment (grgprarup) [inline tests/acceptance/features/apiAuthWebDav/webDavDELETEAuthOC10Issue40144.feature:1]:
```suggestion
@api @notToImplementOnOCIS
```

File: tests/acceptance/features/apiAuthWebDav/webDavDELETEAuthOC10Issue40144.feature (hunk 1 above)
Comment (kiranadh1452) [reply tests/acceptance/features/apiAuthWebDav/webDavDELETEAuthOC10Issue40144.feature]: okay

**Review by grgprarup — COMMENTED** [review]
lgtm :+1:

**Review by saw-jan — COMMENTED** [review]
:+1:

**Review by phil-davis — APPROVED** [review]

**Pull request merged**